docs/auditors/mounts.md

```diff
@@ -0,0 +1,139 @@
+| :------------------------ | :--------------------------------------------------------------------------------------- |
+| /proc                     |  Pseudo-filesystem which provides an interface to kernel data structures                                                  |
+| /var/run/docker.sock      |  Unix socket used to communicate with Docker daemon                                   |
+| /                         |  Filesystem's root |
+| /etc                      |  Directory that usually contains all system related configurations files         |
+| /root                     |  Home directory of the `root` user                                             |
+| /var/run/crio/crio.sock   |  Unix socket used to communicate with the CRI-O Container Engine                                                 |
+| /home/admin               |  Home directory of the `admin` user        |
+| /var/lib/kubelet          |  Directory for Kublet-related configuration                                                             |
+| /var/lib/kubelet/pki      |  Directory containing the certificate and private key of the Kublet                                                               |
+| /etc/kubernetes           |  Directory containing Kubernetes related configuration              |
+| /etc/kubernetes/manifests |  Directory containing manifest of Kubernetes components                                                         |
+
+
+## Examples
+
+```
+$ kubeaudit mounts -f /Users/j.courtial/dev/go/src/github.com/kubeaudit/auditors/mounts/fixtures/proc-mounted.yml
+
+---------------- Results for ---------------
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: pod
+    namespace: proc-mounted
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: proc-volume (/proc -> /host/proc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      Mount: proc-volume
+
+```
+
+## Explanation
+
+Mounting some sensitive host paths (like `/etc`, `/proc`, or `/var/run/docker.sock`) may allow a container to access sensitive information from the host like credentials or to spy on other workloads' activity.
+
+These sensitive paths should not be mounted.
+
+Example of a resource which **fails** the `mounts` audit:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  template:
+    spec:
+      containers:
+        - name: container
+          image: scratch
+          volumeMounts:
+            - mountPath: /host/proc
+              name: proc-volume
+      volumes:
+        - name: proc-volume
+          hostPath:
+            path: /proc
+```
+
+## Override Errors
+
+First, see the [Introduction to Override Errors](/README.md#override-errors).
+
+The override identifier has the format `allow-host-path-mount--[mount name]` which allows for each mount to be individually overridden.
```

Comment:
I think the double dash should be a single dash to match the code and examples. Honestly, I kind of like the double dash better but single dash would be consistent with capabilities overrides... What do you think?

Also, I saw some logic to convert the volume name to override label (lowercase and replace underscores with dashes). Maybe you did that to be consistent with the capabilities auditor, but if the character set for volume names is allowed in labels I wouldn't be opposed to just using the volume name in the override label without any conversion. For example if the volume name is `my_Volume` then the override label would be `allow-host-path-mount--my_Volume`. What do you think? (I don't actually know if `my_Volume` is a valid volume name but just for argument sake 😄 )

Reply:
Totally right for both. The double dash was a typo. Regarding the volume name, indeed let's keep things simple and use the volume name as it is 👍